packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -18,14 +18,20 @@ import {
   InternalMonitoring,
   callMonitored,
   createHandlingStack,
+  Omit,
+  tuple,
 } from '@datadog/browser-core'
 import { RumEventDomainContext } from '../domainContext.types'
 import { CommonContext, User, ActionType } from '../rawRumEvent.types'
 import { RumEvent } from '../rumEvent.types'
 import { buildEnv } from './buildEnv'
 import { startRum } from './startRum'
 
-export interface RumInitConfiguration extends InitConfiguration {
+// eslint-disable-next-line local-rules/disallow-side-effects
+const droppedConfigurationOptions = tuple('publicApiKey', 'datacenter')
```

Comment:
I think you can do:

```suggestion
const droppedConfigurationOptions = [ 'publicApiKey' as 'publicApiKey', 'datacenter' as 'datacenter'] 
```

which is a bit verbose but doesn't add anything at runtime.

Reply:
solution: `const droppedConfigurationOptions = ['publicApiKey' as const, 'datacenter' as const]
`